crates/aptos/src/genesis/mod.rs

```diff
@@ -295,7 +297,9 @@ pub fn fetch_genesis_info(git_options: GitOptions) -> CliTypedResult<GenesisInfo
             employee_vesting_start: layout.employee_vesting_start,
             employee_vesting_period_duration: layout.employee_vesting_period_duration,
             consensus_config: OnChainConsensusConfig::default(),
-            execution_config: OnChainExecutionConfig::default(),
+            execution_config: OnChainExecutionConfig::V1(ExecutionConfigV1 {
+                transaction_shuffler_type: TransactionShufflerType::SenderAwareV2(32),
+            }),
```

Comment:
Should this be revert this back to being the `default()`? For anyone wanting to test other onchain execution configs in forge, they should be aware to change here.

Reply:
@ibalajiarun - Good point. The issue is in Forge, there is not a good way to override various configs.. I will look to see if we can do it without overriding it here but lmk if you have any suggestion.